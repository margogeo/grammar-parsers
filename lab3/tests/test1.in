print(f(1))
print(__g__(2))
bb = int(input())

ab = f(1) + __g__(2) + f(__g__(bb))
print(ab)