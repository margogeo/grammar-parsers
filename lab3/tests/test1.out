print(f(1))
print(__g__(2))
OIOI0IOI0 = int(input())

I11001010=2874+2983
O0OI0IOI0 = f(1) + __g__(2) + f(__g__(OIOI0IOI0))
print(O0OI0IOI0)
IOI10OI101OI0=2064+8483
